### PR TITLE
Serverless Terms Blog Post Initial Draft

### PR DESCRIPTION
**Pull request by techmaharaj**
Initial draft for the serverless terms blog post.

### CODE REVIEW

File: content/en/blog/8-must-know-serverless-terms.md [hunk 1]
```diff
@@ -0,0 +1,158 @@
++++
+title = "8 Must Know Serverless Terms For A Developer"
```
Comment (sanketsudake) [inline content/en/blog/8-must-know-serverless-terms.md:2]: Title should be
`8 Must Know  Terms For A Serverless Developer`

File: content/en/blog/8-must-know-serverless-terms.md [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+date = "2022-03-01T11:30:34+05:30"
+author = "Atulpriya Sharma"
+description = "Planning to start your cloud native journey? Here are 8 Must Know Serverless Terms For A Developer."
```
Comment (sanketsudake) [inline content/en/blog/8-must-know-serverless-terms.md:5]: Are you building a serverless application? Here are 8 must know terms for a serverless developer

File: content/en/blog/8-must-know-serverless-terms.md [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+categories = ["General"]
+type = "blog"
+images = ["/images/featured/serverless-terms.png"]
++++
+
+I’ve been a mobile developer for most of my professional career.
+The transition from Java to Android wasn’t a big deal. However, over the years the entire technological landscape has transformed, thanks to serverless and microservices. Many of the mobile apps that I developed have a serverless component running somewhere on the cloud.
```
Comment (sanketsudake) [inline content/en/blog/8-must-know-serverless-terms.md:12]: These statements doesnt fit in the context.

The transition from Java to Android wasn’t a big deal. However, over the years the entire technological landscape has transformed, thanks to serverless and microservices.

May be something of this sort

I’ve been a mobile developer for most of my professional career. Many mobile application has shifted their backend from microservices to serverless. As I focused on building serverless applications, this post highlights terms every serverless developer should know.

File: content/en/blog/8-must-know-serverless-terms.md [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+
+What that meant is that a lot of us developers had to get a hang of the serverless world.
+So to help all of you looking to get started with the serverless journey, here are the **8 Must Know Serverless Terms** you should know as a developer.
```
Comment (sanketsudake) [inline content/en/blog/8-must-know-serverless-terms.md:15]: We can merge the above two paras and keep the intro short.

File: content/en/blog/8-must-know-serverless-terms.md [hunk 5]
```diff
@@ -0,0 +1,158 @@
...
+
+{{< figure src="/images/featured/serverless-terms.png" alt="Planning to start your cloud native journey? Here are 8 Must Know Serverless Terms For A Developer." height="600" width="1000">}}
+
+## Serverless Terms You Should Know
+
+Serverless is the present and future of the software industry.
```
Comment (sanketsudake) [inline content/en/blog/8-must-know-serverless-terms.md:21]: This is a bold statement. 

We can say instead

Serverless is popular amongst developers for ease of building and deployment.

File: content/en/blog/8-must-know-serverless-terms.md [hunk 6]
```diff
@@ -0,0 +1,158 @@
...
+Irrespective of what type of an IT professional you are, you would definitely have to interact with some serverless or cloud components in the future.
```
Comment (sanketsudake) [inline content/en/blog/8-must-know-serverless-terms.md:22]: Lets consider persona for this blog as serverless developers only.

A lot of things change when moving to serverless and building architecture for serverless applications.

Comment (sanketsudake) [reply content/en/blog/8-must-know-serverless-terms.md:22]: So high level persona for this blog is `serverless architect` not even `serverless developer.`
but serverless developer is wide term  than architect so we can go with developer keyword.

If we analyze all terms these are must know for a serverless architect. developer may or may not necessarily know them.

Comment (sanketsudake) [reply content/en/blog/8-must-know-serverless-terms.md:22]: I am fine with either architect or developer based on seo analysis.

**Review by sanketsudake — CHANGES_REQUESTED** [review]

File: content/en/blog/8-must-know-serverless-terms.md (hunk 2 above)
Comment (techmaharaj) [reply content/en/blog/8-must-know-serverless-terms.md]: Done.

File: content/en/blog/8-must-know-serverless-terms.md (hunk 5 above)
Comment (techmaharaj) [reply content/en/blog/8-must-know-serverless-terms.md]: Understood, done.

File: content/en/blog/8-must-know-serverless-terms.md (hunk 6 above)
Comment (techmaharaj) [reply content/en/blog/8-must-know-serverless-terms.md]: Left it as developer.

**Review by sanketsudake — APPROVED** [review]

**Pull request merged**